packages/osfingermap.yaml

```
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# vim: ft=yaml
+---
+CentOS Linux-6:
```

This is the actual `osfinger` for this platform:

```suggestion
CentOS-6:
```